backend/models/vendor.js

```diff
@@ -3,14 +3,29 @@ const ClientBase = require("./client");
 
 const Schema = mongoose.Schema;
 
+var address = {
+  addressName: { type: String },
+  name: { type: String },
+  addressLine: { type: String },
+  city: { type: String },
+  state: { type: String },
+  zipCode: { type: String },
+  phone: { type: String },
+};
+
+var location = {
+  location: { type: String },
+};
+
 var vendorSchema = new Schema(
   {
     companyName: { type: String },
     companyDomainName: { type: String },
     aboutCompany: { type: String },
+    products: { type: String },
```

Comment:
This field is also obsolete, Products will be a separate model and we will use MongoDB queries to get products a vendor sells

Reply:
Removed.